changelog.md

```diff
@@ -172,6 +172,8 @@ proc mydiv(a, b): int {.raises: [].} =
   (likewise with other hints and warnings), which is consistent with all other bool flags.
   (since 1.3.3).
 - `nim doc -r main` and `nim rst2html -r main` now call openDefaultBrowser
+- `nim doc foo/main` now outputs to `foo/main.html`, consistently with other commands. Use
+  `--usenimcache` for example to output instead under $nimcache.
```

Comment:
```suggestion
- `nim doc foo/main` now outputs to `foo/main.html` by default, consistently with other commands. Use `--outdir:.` to restore old behavior.
```

Users don't really care about `--useNimcache` in a change log, they would care more about how to get the old semantics back.

Reply:
well `--useNimcache` is new (and is well suited for things like `nim doc` since it avoids clobbering sources); in any case, I've now also added how to get back previous behavior.